src/components/images/imageLoader.js

```diff
@@ -21,6 +21,37 @@ define(['lazyLoader', 'imageFetcher', 'layoutManager', 'browser', 'appSettings',
             return;
         }
 
+        try {
+            var blurHash = elem.getAttribute("data-blurhash");
+            if (blurHash && blurhash.isBlurhashValid(blurHash)) {
+                var width = 85; // TODO: get correct dimensions
+                var height = 128;
```

Comment:
Any idea how I could get the dimensions for the element here? innerWidth and innerHeight are still NaN by the time this code runs.

Reply:
`innerWidth`/`innerHeight` belongs to `Window`
What about `offsetWidth`/`offsetHeight` (or `clientWidth`/`clientHeight`)?

Reply:
Oops, you're right. That was dumb of me!